libs/design/src/atoms/form/select/select/select.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, ViewEncapsulation, DoCheck, Input, HostBinding } from '@angular/core';
+import { FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'select[daff-select]',
+  template: '<ng-content></ng-content>',
+  styleUrls: ['./select.component.scss'],
+  host: {
+    'class': 'daff-select'
+  },
+  encapsulation: ViewEncapsulation.None,
+})
+
+export class DaffSelectComponent implements DoCheck {
+
+  //todo: will be removed once we upgrade to Angular 8, where markAllAsTouched will be a method on FormControl.
+  @Input() formSubmitted: boolean;
+  @Input() formControl: FormControl;
+  @HostBinding('class.daff-select__error') isError: boolean;
+  @HostBinding('class.daff-select__valid') isValid: boolean;
+
+  ngDoCheck() {
+    if(this.formControl) {
+      this.isError = this.formControl.errors && (this.formControl.touched || this.formSubmitted);
+      this.isValid = !this.formControl.errors && this.formControl.touched;
+    }
+  }
```

Comment:
I don't think 19-27 needs to be added for this PR, but if you wanted to combine `daff-select` and `daff-select-validator`, this needs to be tested.

Reply:
fixed in 83fb883